onnxruntime/core/session/inference_session.h

```diff
@@ -534,6 +534,8 @@ class InferenceSession {
 #endif
 
   // used to hold the ModelProto parsed in an applicable ctor to be used while calling parameter-less Load()
-  std::unique_ptr<ONNX_NAMESPACE::ModelProto> model_proto_;
+  ONNX_NAMESPACE::ModelProto model_proto_;
+
+  bool model_proto_parsed_ = false;
 };
```

Comment:
nit: name could be more generic, like 'model_loaded_'

Reply:
will fix.

Reply:
Fixed.